go.mod

```diff
@@ -200,6 +200,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v0.4.1
+replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v1.7.4-0.20221107225611-7cbc562f7437
```

Comment:
https://github.com/iotexproject/go-ethereum/commit/7cbc562f74375a00db6e4bcbc6c6d6859b7a6df2

once verified fullsync, it will become a tag/version

Reply:
it seems i cannot create a PR using this commit ( atari branch against v0.4.1 tag)